src/style/utils/merge-deep.spec.js

```diff
@@ -41,4 +41,10 @@ describe('mergeDeep', () => {
   it('merges the objects but does not create a new key when not passed an object', () => {
     expect(mergeDeep(base, undefined)).toEqual(base);
   });
+
+  it('is not mutative', () => {
+    mergeDeep(base, toMerge);
+    const secondObject = mergeDeep(base, objectWithArray);
+    expect(secondObject).toEqual({ a: 1, b: { nestedB: 1 }, c: ['inArray'] });
```

Comment:
should the actual test be that `base` is still the same? as that was the thing that was getting mutated.

Reply:
You might just be right.